### DISCUSSION TITLE
If there are multiple y data, can they be included in the analysis?

### DISCUSSION THREAD

**Discussion by boozesounds** [ourownstory/neural_prophet#367] [Q&A - get help using NeuralProphet]
a is y and we want to analyze b and c together.
![5444](https://user-images.githubusercontent.com/35713030/126428591-a7476e16-1ca9-40c2-8887-4deb989d010c.jpg)

But if I use add lagged regressor
![ccccc](https://user-images.githubusercontent.com/35713030/126429028-f508c507-3538-498f-9b0c-85d320308f9b.jpg)

The contents of the csv file are not included, but the existing y data is written as it is. What should I do?

**Comment by ourownstory** [comment]
hi @boozesounds Do you mind explaining in more detail what you are trying to do here?

**Reply by boozesounds** [reply]
Exogenous variables of a are b and c. Since (b,c) is an exogenous variable related to (a) data, I am sure that analyzing b,c together will increase the prediction accuracy. But I don't know how to analyze the exogenous variables b and c together when predicting the future of a.
